packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -373,4 +413,17 @@
     <T.Panel><DummyPlaceholder @text="Content nine" @height="50" @background="#eee" /></T.Panel>
     <T.Panel><DummyPlaceholder @text="Content ten" @height="50" @background="#eee" /></T.Panel>
   </Hds::Tabs>
+
+  <h5 class="dummy-h6">Call a passed function when a tab is clicked</h5>
+  <p class="dummy-paragraph">Logs the tab id to the console</p>
+
+  <Hds::Tabs @onClickTab={{this.logClickedTab}} as |T|>
+    <T.Tab>One</T.Tab>
+    <T.Tab>Two</T.Tab>
+    <T.Tab>Three</T.Tab>
+
+    <T.Panel><DummyPlaceholder @text="Content one" @height="50" @background="#eee" /></T.Panel>
+    <T.Panel><DummyPlaceholder @text="Content two" @height="50" @background="#eee" /></T.Panel>
+    <T.Panel><DummyPlaceholder @text="Content three" @height="50" @background="#eee" /></T.Panel>
+  </Hds::Tabs>
```

Comment:
Now that you've added the same example in the "How to use" section, I would remove it from the "Showcase" section (which is mainly visual, and in this case the difference with other examples is not visible)

Reply:
Ok

Reply:
I just did the reverse by accident. Fixing and pushing up.

I included the example in the showcase since you can click on the tabs and see the console log message which demonstrates the feature. If I delete that, should I also delete the demo function I included in tavs.js?

```
@action
  logClickedTab(event) {
    const tabId = event.target.id;
    console.log(`Tab with ID "${tabId}" clicked!`);
  }
```

Reply:
I undeleted the example code but left in the example in the Showcase for now. To me I think it's a potentially helpful demo but let me know if you'd rather I delete both the Showcase example and the `logClickedTab` demo JS.

Reply:
Going to merge as is for now and can delete these later if you wish

Reply:
@KristinLBradley let's discuss tomorrow